.github/pull_request_template.md

```diff
@@ -7,11 +7,6 @@ This PR template is here to help ensure you're setup for success:
 **JIRA Ticket:**
 [BFD-123456](https://jira.cms.gov/browse/BFD-123456)
```

Comment:
Consider simplifying this. This repository is configured to link DASG projects, e.g. `BFD-123` -> becomes BFD-123 without the markdown. I'll leave that up to as to whether we remove the markdown formatting for the jira issue link 😉 
```suggestion
BFD-XXXX
```